tests/template-tester/run.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2021 VMware
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package template_tester
```

Comment:
Should be a [single word](https://go.dev/doc/effective_go#names)

could easily be "testing"

Also I think that the "tester" is a product, so should be in pkg.  eg: `pkg/testing`